components/filter/filter.js

```diff
@@ -710,6 +732,24 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 		this._activeDimensionKey = null;
 	}
 
+	_handleDimensionLoadMore(e) {
+		const dimensionKey = e.target.parentNode.id.slice(SET_DIMENSION_ID_PREFIX.length);
+		const dimension = this._getDimensionByKey(dimensionKey);
+		const searchCompleteCallback = this._getSearchCallback(dimension);
```

Comment:
```suggestion
		const applySearch = this._getSearchCallback(dimension);
```
This might make it a bit clearer how it's being used here? Basically after load more is complete, we trigger a search to update visibility.  It's kinda an odd workflow - it works because we _know_ they're using manual search.  But I do think it's cleaner than sending a load-more event and search event separately.